services/core-web/src/setupTests.js

```diff
@@ -4,14 +4,20 @@ import path from "path";
 
 require("jest-localstorage-mock");
 
-global.REQUEST_HEADER = require(path.resolve(__dirname, "../common/utils/RequestHeaders.js"));
-
 Enzyme.configure({ adapter: new Adapter() });
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+global.REQUEST_HEADER = require(path.resolve(__dirname, "../common/utils/RequestHeaders.js"));
+
 global.requestAnimationFrame = (callback) => {
   setTimeout(callback, 0);
 };
 
+jest.mock("react-lottie", () => ({
+  __esModule: true,
+  default: "lottie-mock",
+}));
+
```

Comment:
Not a problem with your PR, but this gives me an idea, would this actually be the best place to set up a mock for useFeatureFlag instead of individual files? Some like useParams we can't quite generalize but I think we always want the isFeatureEnabled function to return true.